.circleci/base.yml

```diff
@@ -12,7 +12,7 @@ commands:
 jobs:
   fast-checks:
     docker:
-      - image: skiplabs/skip:latest
+      - image: skiplabs/skip:llvm16
```

Comment:
Do we want to tag docker images by llvm version?

Reply:
If we don't plan on officially supporting and/or using more than one version, probably not.
Here it was just for testing